### PR TITLE
Fix #254 :  Up to date README.md

### LINKED ISSUE #254: [Docs] The backend readme file is not up to date
In the README.md missing a part for build images in server directory 
``` 
<scratchy/server> sudo make 
```

### PR DESCRIPTION
**Pull request by yannis-mlgrn**
Fixes : #254 
In this issue I have added in server README.md file : 
-  the command line to build images
-  <s> sudo prefix before some command like `docker-compose up -d` who doesn't run if you don't have sudo privileges </s>

### CODE REVIEW

File: server/README.md [hunk 1]
```diff
@@ -17,21 +17,26 @@ Start mongodb:
 * prerequisites:
   * install docker: see [Install Docker Engine](https://docs.docker.com/engine/install/)
   * install docker-compose: see [Install Docker-compose](https://docs.docker.com/compose/install/) or use the package management of your GNU/Linux distribution (Ubuntu for example)
+
+* build images in server directory
+```sh
+<scratchy/server>$ sudo make install
```
Comment (viandoxdev) [inline server/README.md:23]: there is no install target in the Makefile, this should be `make build`

File: server/README.md [hunk 2]
```diff
@@ -78,7 +83,7 @@ scratchy-server
 
 * Restart mongodb using docker-compose:
 ```sh
-<scratchy/docker>$ docker-compose up -d
+<scratchy/docker>$ sudo docker-compose up -d
```
Comment (viandoxdev) [inline server/README.md:86]: sudo isn't really needed it depends on the user's settings, this should be addressed with some kind of note ( > blablabla ) saying that sudo should be used if it fails without.
(I only write this on this line but this holds for every command with sudo)

**Review by viandoxdev — CHANGES_REQUESTED** [review]
(see review)

File: server/README.md (hunk 2 above)
Comment (viandoxdev) [reply server/README.md:86]: For example i didn't need to use sudo on my computer

**Comment by yannis-mlgrn** [comment]
fixes

File: server/README.md (hunk 2 above)
Comment (yannis-mlgrn) [reply server/README.md:86]: fixes

**Comment by yannis-mlgrn** [comment]
problem

**Pull request closed without merging**